### ISSUE TITLE
Config?

### ISSUE THREAD

**Issue by virtualxtetris** [FriederHannenheim/PortableNetherFabric#1]
Hey man i have a problem with this mod. my start world is the moon from a other mod and now i have your mod he can teleport me to the nether but when i teleport me back he teleports me to the overworld and that is wrong is there a whay that he port me to the last location like a config funktion or something

**Comment by FriederHannenheim** [comment]
No. You shouldn't be able to teleport yourself from the moon anyway. You should only be able to teleport to the nether from the overworld. Idk if nether portals work on the moon with the mod you are using but I initially wanted teleportation to only happen between the overworld and the nether. Because nether portals don't work in the end.

**Comment by virtualxtetris** [comment]
i can use the nether portal from the moon but i get not teleportet back to the moon 
can i not change the teleport comand what the ther portal use to go back to the moon ?

**Comment by virtualxtetris** [comment]
or have you a idee how i can give a item like a cake a minecraft teleport command ?
